### DISCUSSION TITLE
How to import rules?

### DISCUSSION THREAD

**Discussion by alexholox** [evilsocket/opensnitch#576] [Q&A]
.

**Comment by gustavo-iniguez-goya** [comment]
Hi Alex,

The way to do it right now is by copying `/etc/opesnitchd/rules/*.json` files and move the rules to the other computers, via scp, ftp, etc.

We can add options to apply a set of rules to a particular node, it was asked some months ago.

**Comment by gustavo-iniguez-goya** [comment]
Added option for applying a rule or set of rules to a node.
Having multiple nodes connected:
- Select one or more rows.
- Right click on the selection.
- Select "Apply to -> <node>" to apply the rules to the selected node.
